### ISSUE TITLE
Add more expressive print statements to optimization script

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#1862]
When running the optimization script, It's hard to know where in the process we currently are, when the output is mainly pages and pages of:

```
[...]
8461
8462
8463
8464
8465
8466
8467
8468
8469
8470
[...]
```

@shanshanhsieh, @BhargavaKrishnaSreepathi do you know where this is being printed? Do you know what that number means? It would be good if those lines included more information, especially:

- individual X of N
- generation Y of M
- a hint to what the number (e.g. 8470) represents.
- and maybe a projection of time to completion?

**Comment by daren-thomas** [comment]
> and maybe a projection of time to completion?

let's make the projection a separate issue

**Comment by gabriel-happle** [comment]
@BhargavaKrishnaSreepathi fixed this. It was merged with the epic #1600
